Snowplow/SPSubject.m

```diff
@@ -110,22 +123,60 @@ - (void) setDomainUserId:(NSString *)duid {
+    [_platformDict addValueToPayload:[SPUtilities getOSVersion]         forKey:kSPPlatformOsVersion];
+    [_platformDict addValueToPayload:[SPUtilities getDeviceVendor]      forKey:kSPPlatformDeviceManu];
+    [_platformDict addValueToPayload:[SPUtilities getDeviceModel]       forKey:kSPPlatformDeviceModel];
 #if TARGET_OS_IPHONE
     [self setMobileDict];
 #endif
 }
 
 - (void) setMobileDict {
-    [_platformDict addValueToPayload:[SPUtils getCarrierName]       forKey:kSPMobileCarrier];
-    [_platformDict addValueToPayload:[SPUtils getOpenIdfa]          forKey:kSPMobileOpenIdfa];
-    [_platformDict addValueToPayload:[SPUtils getAppleIdfa]         forKey:kSPMobileAppleIdfa];
-    [_platformDict addValueToPayload:[SPUtils getAppleIdfv]         forKey:kSPMobileAppleIdfv];
-    [_platformDict addValueToPayload:[SPUtils getNetworkType]       forKey:kSPMobileNetworkType];
-    [_platformDict addValueToPayload:[SPUtils getNetworkTechnology] forKey:kSPMobileNetworkTech];
+    [_platformDict addValueToPayload:[SPUtilities getCarrierName]       forKey:kSPMobileCarrier];
+    [_platformDict addValueToPayload:[SPUtilities getOpenIdfa]          forKey:kSPMobileOpenIdfa];
+    [_platformDict addValueToPayload:[SPUtilities getAppleIdfa]         forKey:kSPMobileAppleIdfa];
+    [_platformDict addValueToPayload:[SPUtilities getAppleIdfv]         forKey:kSPMobileAppleIdfv];
+    //[_platformDict addValueToPayload:[SPUtilities getNetworkType]       forKey:kSPMobileNetworkType];
+    //[_platformDict addValueToPayload:[SPUtilities getNetworkTechnology] forKey:kSPMobileNetworkTech];
+}
+
+// Geo-Location Dictionary
+
+- (void) setGeoDict {
+    _geoLocationDict = [[NSMutableDictionary alloc] init];
+}
+
+- (void) setGeoLatitude:(float)latitude {
+    [_geoLocationDict setObject:[NSNumber numberWithFloat:latitude] forKey:kSPGeoLatitude];
+}
+
+- (void) setGeoLongitude:(float)longitude {
+    [_geoLocationDict setObject:[NSNumber numberWithFloat:longitude] forKey:kSPGeoLongitude];
+}
+
+- (void) setGeoLatitudeLongitudeAccuracy:(float)latitudeLongitudeAccuracy {
+    [_geoLocationDict setObject:[NSNumber numberWithFloat:latitudeLongitudeAccuracy] forKey:kSPGeoLatLongAccuracy];
+}
+
+- (void) setGeoAltitude:(float)altitude {
+    [_geoLocationDict setObject:[NSNumber numberWithFloat:altitude] forKey:kSPGeoAltitude];
+}
+
+- (void) setGeoAltitudeAccuracy:(float)altitudeAccuracy {
+    [_geoLocationDict setObject:[NSNumber numberWithFloat:altitudeAccuracy] forKey:kSPGeoAltitudeAccuracy];
+}
+
+- (void) setGeoBearing:(float)bearing {
+    [_geoLocationDict setObject:[NSNumber numberWithFloat:bearing] forKey:kSPGeoBearing];
+}
+
+- (void) setGeoSpeed:(float)speed {
+    [_geoLocationDict setObject:[NSNumber numberWithFloat:speed] forKey:kSPGeoSpeed];
+}
+
+- (void) setGeoTimestamp:(NSInteger)timestamp {
+    //[_geoLocationDict setObject:[NSNumber numberWithInt:(int)timestamp] forKey:kSPGeoTimestamp];
```

Comment:
This is to remove conflict with schemas, my fix was to remove all conflicts in one commit that I can later delete.